### PR TITLE
¡ɐpqɯɐl ʎɹǝʌ 'lɐuoᴉʇɔunɟ ǝɹoW

### PR DESCRIPTION
**Pull request by DaveDuck321**
Pls test, filter >> map

Full patch or nothing. thanks.

Your welcome,

### CODE REVIEW

**Comment by WeixuanZ** [comment]
Thx bro

File: src/prediction.py [hunk 1]
```diff
@@ -21,14 +22,15 @@
...
-income_scalar = MinMaxScaler(census_df, 'household_income')
-home_scalar = MinMaxScaler(census_df, 'home_value')
+scalars = list(map(
+    partial(MinMaxScaler, census_df),
+    ['population', 'household_income', 'home_value']
+))
 
 # normalisation
-list(map(
+len(filter(
```
Comment (WeixuanZ) [inline src/prediction.py:31]: No filters plz 👻

**Review by WeixuanZ — CHANGES_REQUESTED** [review]

File: src/prediction.py (hunk 1 above)
Comment (DaveDuck321) [reply src/prediction.py]: Fixed! more functions make it better

**Review by WeixuanZ — APPROVED** [review]
ಠ_ರೃ

**Pull request merged**